app/src/main/java/com/android/sample/ui/activity/EditActivity.kt

```diff
@@ -615,7 +615,7 @@ fun EditActivityScreen(
                         Log.e("EditActivityScreen", "Failed to remove images: ${error.message}")
                       })
 
-                  navigationActions.navigateTo(Screen.OVERVIEW)
```

Comment:
Changed the navigation destination from Screen.OVERVIEW to Screen.PROFILE after the image removal operation. LGTM!